### PR TITLE
qm-gui-about

### PR DESCRIPTION
**Pull request by Mashintime**
- Added about modal
- Header redesign
- Heard station table tweaks
- Stats can be enabled independent of explorer reporting
- Chat messages have a lower UUID lessening protocol overhead

### CODE REVIEW

File: gui/sock.js [hunk 1]
```diff
@@ -542,8 +542,8 @@ exports.sendMessage = function(dxcallsign, mode, frames, data, checksum, uuid, c
     //data = utf8.encode(data)
 
     //socketLog.info(data)
-
-    
+    //shorten uuid
```
Comment (DJ2LS) [inline gui/sock.js:545]: This comments can be removed?

**Review by DJ2LS — APPROVED** [review]
Looks good so far from code point of view need to do a short test to check the gui

**Comment by DJ2LS** [comment]
Gui looks good, thanks @Mashintime 

Suggestions: 
The dropdown field for settings, explorer and so on - I'm not sure if it's too close to start/stop the tnc. Also about the colors. Maybe we should use a grey style für the "chat" as well. Maybe "grey + outline" and moving the Dropdown arrow to the right and making it slightly bigger. Maybe with padding class ( p-2 ) or so. What do you think?
For the dropdown elements I suggest moving the icons to the left and using icons for all elements.

**Comment by DJ2LS** [comment]
note: failing ctest for python 3.12-dev can be ignored

**Comment by DJ2LS** [comment]
image

@Mashintime , what about this?

**Comment by Mashintime** [comment]
Looks good to me.  Commit if you'd like, otherwise can implement later today @DJ2LS

**Comment by DJ2LS** [comment]
@Mashintime I would appreciate, if you could implement it if you like. Just take the code from comment above.

**Comment by Mashintime** [comment]
@DJ2LS no problem!

File: gui/sock.js (hunk 1 above)
Comment (Mashintime) [reply gui/sock.js]: They've been cleaned up.

**Comment by DJ2LS** [comment]
@Mashintime I'm ready for merging this one. So we can test from main together with the other improvements on ls-arq

**Comment by Mashintime** [comment]
@DJ2LS looks good to me

**Pull request merged**